code-challenges.test.js

```diff
@@ -48,10 +97,41 @@ const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
 const randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
-
+//
+// describe("capitalizeMachine", () => {
+//   it("take in an array and make all words capitalized", () => {
+// expect(capitalized()).toEqual(["array of all words capitalized"])
```

Comment:
same here..hard code what you expect to see. If there are three variables, then you should have three expect statements